Jenkinsfile

```diff
@@ -56,6 +58,7 @@ pipeline {
                 sh 'npm run build'
                 sh 'mkdir deployment && mv dist deployment && mv docker deployment'
```

Comment:
If you get a `client` folder and a `server` folder in dist, can you `oc start-build` the client with the `client` folder **only** and the server with the `server` folder **only**?

Also, for the server part, are the node_modules dependencies included in the bundle? If not, you'll have to pass the `package.json` and do a `npm i --production`

Reply:
That's a really great suggestion, thanks!